src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -27,20 +27,16 @@
 
       <h1>User Profile Pages</h1>
 
-      <% String profilePageUserName = (String) request.getAttribute("username");
+      <%
       String currentUser = (String) request.getSession().getAttribute("user");
+      String username = (String) request.getAttribute("profilePage");
 
-      if (profilePageUserName == "") { %>
```

Comment:
Never use == to compare strings. Use String.equals()

Reply:
yes thank you! I deleted this line